hs/src/Reach/Simulator/Core.hs

```diff
@@ -96,20 +96,28 @@ data Ledger = Ledger
   }
   deriving (Eq)
 
+nwToken :: Token
+nwToken = -1
+
+simContract :: Account
+simContract = -1
+
+simContractAmt :: Balance
+simContractAmt = 4444
```

Comment:
The contract will start with nothing, FWIW. And it doesn't exist until after the first message, so it might not be the "lowest" one. Because of remote objects, you're going to have a way of creating new contracts out there in the world too